packages/block-editor/src/components/global-styles/style.scss

```
@@ -58,6 +58,11 @@
 	}
 }
 
+.block-editor-global-styles-filters-panel__dropdown {
+	border: 1px solid rgba(0, 0, 0, 0.1);
```

```suggestion
	border: 1px solid $gray-300;
```

I think there should be no need to use the RGBA value.